packages/core/src/components/navbar/_navbar.scss

```diff
@@ -61,10 +61,17 @@ $dark-navbar-background-color: $dark-gray5;
 
   &.pt-dark,
   .pt-dark & {
-    box-shadow: inset $pt-dark-elevation-shadow-1;
     background-color: $dark-navbar-background-color;
   }
 
+  &.pt-dark {
+    box-shadow: inset $pt-dark-elevation-shadow-1;
+  }
+
+  .pt-dark & {
```

Comment:
going to at least need a code comment explaining how these two cases are different